app/resources/numismatics/coins/orangelight_coin_builder.rb

```diff
@@ -60,7 +60,8 @@ def document_coin_hash
         size_s: decorator.size,
         technique_s: decorator.technique,
         weight_s: decorator.weight,
-        holdings_1display: holdings_hash
+        holdings_1display: holdings_hash,
+        accession_info_s: decorator.accession_person
```

Comment:
Should we call it `donor` here? We sort of lose the fact that it's a person in this step.

Reply:
👍 That's a good idea.

Reply:
Updated.